scripts/deployment/bridges/optimistic/test/messenger_sepolia_sepolia_governor.js

```diff
@@ -27,15 +27,20 @@ async function main() {
     const CDMProxy = new ethers.Contract(CDMProxyAddress, CDMProxyABI, sepoliaProvider);
 
     // Test deployed OptimismMessenger address on optimisticSepolia
-    const optimismMessengerAddress = "0x670Ac235EE13C0B2a5065282bBB0c61cfB354592";
+    //const optimismMessengerAddress = "0xeDd71796B90eaCc56B074C39BAC90ED2Ca6D93Ee"; // original
+    const optimismMessengerAddress = "0x670Ac235EE13C0B2a5065282bBB0c61cfB354592"; // payable process on L2
```

Comment:
Correcting setup on Optimism as we made sure this is the correct one that works on Base, and corresponds to our requirements.